bears/general/IndentationBear.py

```diff
@@ -399,7 +403,8 @@ def get_first_unindent(indent,
     :param start_line:      The line from where to start searching for unindent.
     :param annotation_dict: A dictionary containing sourceranges of all the
                             strings and comments within a file.
-    :param encapsulators:
+    :param encapsulators:   A tuple of sourceranges of all occurances of
```

Comment:
sorry for being picky... but you correctly cased SourceRanges in line 300, maybe do that here also? also "occurEnces" I think.

Also: what is a "SourceRange of occurences of encapsulators", that souds irritating. You mean something like "SourceRanges of code regions trapped in between a matching pair of encapsulators" right? maybe there is a nicer way to say that.